rcmd/configure.go

```diff
@@ -80,3 +86,16 @@ func configureEnv(sysEnvVars []string, rs RSettings, pkg string) []string {
 
 	return envVars
 }
+
+
+// Returns a constant set of env vars to be hidden in logs.
+// Calling function may pass in additional values to include in the censor list.
+func censoredEnvVars(add []string) []string {
+	censoredVars := []string{"GITHUB_TOKEN", "GITHUB_PAT", "GHE_TOKEN", "GHE_PAT", "AWS_ACCESS_KEY_ID", "AWS_SECRET_KEY",}
```

Comment:
it would make more sense to have censoredEnvVars be a map - that way you get instant lookup and do not need to use go-funk to scan the array each time.